R/flea_franchises.R

```diff
@@ -16,7 +16,7 @@
 #' @export
 
 ff_franchises.flea_conn <- function(conn) {
-  x <- fleaflicker_getendpoint("FetchLeagueStandings", league_id = conn$league_id, sport = "NFL") %>%
+  x <- fleaflicker_getendpoint("FetchLeagueStandings", season = conn$season, league_id = conn$league_id, sport = "NFL") %>%
```

Comment:
probably should have been indented like this in the first place 😅 
```suggestion
  x <- fleaflicker_getendpoint(
    "FetchLeagueStandings", 
    season = conn$season, 
    league_id = conn$league_id, 
    sport = "NFL"
  )
```

Reply:
forgot a pipe here when I made the suggestion, whoops